browser-test/src/support/admin_programs.ts

```diff
@@ -522,9 +522,29 @@ export class AdminPrograms {
   }
 
   async createNewVersion(programName: string) {
+    await this.createNewVersionWithFlag(programName, false)
+  }
+
+  async createNewVersionWithReadOnlyViewEnabled(programName: string) {
+    await this.createNewVersionWithFlag(programName, true)
+  }
+
+  async createNewVersionWithFlag(
```

Comment:
**issue (blocking):** `WithFlag` is not a readable or standard naming scheme, you don't need it at all since you can overload the method name and provide method docs to make it clear what the method does.